README.md

```diff
@@ -67,7 +93,7 @@ string of a JSON dictionary. For example:
 
 ## Running Python unit/integration tests
 
-To run Python unit/integration tests, follow the instructions [here](./README_tests.md)
+To run Python unit/integration tests, follow the instructions [here](https://github.com/leaf-ai/neuro-san/blob/main/README_tests.md)
```

Comment:
Use links to GitHub instead of local links that peter out to nowhere when viewed from PyPi.